docs/docs/tutorial/jsonnet.mdx

```diff
@@ -286,12 +286,28 @@ clusters just work.
 This allows us to make sure that you will never accidentally apply to the wrong
 cluster.
 
-> **Note**: Tanka won't create the namespace for you. Either include it in
-> Jsonnet, or with the following command:
+> **Note**: Tanka won't create the namespace for you -- you need to include it in
+> Jsonnet:
+> ```jsonnet
+> {
+>   "my_namespace": {
```

Comment:
This is Jsonnet, keys don‘t need quoting

Reply:
Fixed. To be honest, I was lazy here, and just grabbed YAML namespace definition and converted it via a YAML to JSON converter O:-) . Forgot about jsonnet details.